returnn/tf/util/data.py

```diff
@@ -2973,9 +2990,10 @@ def get_full_name(self):
       return "%s/%s" % (scope_name, self.name)
     return self.name
 
-  def get_dim_tag(self, axis):
+  def get_dim_tag(self, axis, unique_spatial_dims=False):
```

Comment:
I don't really understand this change. What do you need this for? This behavior here in this function seems counter-intuitive to me. This function should just do what it says, i.e. return the dim tag for an axis.

Reply:
In pytorch-to-returnn, I use it to obtain a dim tag where multiple spatial axes get dim tags with different names so that they can be kept apart. Would you prefer to keep this only on pytorch-to-returnn side?

Reply:
Such logic should be implemented in pytorch-to-returnn but not here.

Also, it's actually wrong. It must be the same `DimensionTag` object. That's the idea of dimension tags. If the dimensions are the same, the dimension tags are the same.

The only way to different dimensions then is by `stag-single`.

Reply:
I did this on pytorch-to-returnn side now and removed this part here.